examples/scale-control-directive/index.html

```diff
@@ -3,21 +3,18 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>YAGA - Leaflet-NG2 | Scale-Control Example</title>
+    <title>YAGA | leaflet-ng2 | Scale-Control Example</title>
     <script src="bundle.js"></script>
     <link rel="stylesheet" href="../../node_modules/bootstrap/dist/css/bootstrap.min.css" />
     <link rel="stylesheet" href="../../node_modules/bootstrap/dist/css/bootstrap-theme.min.css" />
     <link rel="stylesheet" href="../../node_modules/leaflet/dist/leaflet.css" />
     <style>
         .yaga-map {
-            min-height: 300px;
+            min-height: 260px;
```

Comment:
but here we should add the default style instead! Look at [map-component example html on line 11](https://github.com/yagajs/leaflet-ng2/blob/develop/examples/map-component/index.html#L11).